### ISSUE TITLE
Is anyone working on this??? This should be the basis.... work on it so that woocommerce can handle large product databases and metadata

### ISSUE THREAD

**Issue by DanielCyran** [woocommerce/woocommerce-product-tables-feature-plugin#210]
Is anyone working on this??? This should be the basis.... work on it so that woocommerce can handle large product databases and metadata

**Comment by kevin25** [comment]
They abandoned this juice and just going around and around

**Comment by rpldom** [comment]
Why this is abandoned? I guess performance is not priority for Woo team?

**Comment by kevin25** [comment]
> Why this is abandoned? I guess performance is not priority for Woo team?

I don't think so. They have business as well

**Comment by ash-f** [comment]
Unless this will be built-in, Woo is not suitable for a shop with wide selection of products.
Can't compete with Shopify or other ec systems.

**Comment by kevin25** [comment]
Yes, they should make it like HPOS. Option for shop owner to use it. This will make it unbeatable.
